### DISCUSSION TITLE
pi-Base community call Fri 2024 Sep 6 at 4pm Eastern

### DISCUSSION THREAD

**Discussion by StevenClontz** [pi-base/data#715] [Announcements]
Agenda:

- discuss https://github.com/pi-base/data/pull/658
- locally Euclidean or locally $n$-Euclidean? https://github.com/pi-base/data/pull/706
- How much context preferred for adding new properties? See https://github.com/pi-base/data/issues/696
- discuss https://github.com/pi-base/data/issues/739

Zoom link: https://southalabama.zoom.us/j/93833246214

**Comment by ccaruvana** [comment]
Throwing this here in case we want to add to the agenda since we talked a little bit about it: https://github.com/pi-base/data/issues/696

I will add some comments regarding this issue in the issue thread.

**Reply by StevenClontz** [reply]
Cool. You can edit my agenda above to add it yourself if you like.

**Comment by ccaruvana** [comment]
It seems that the Carolinas seminar will be meeting on the 30th. Should we drop this back a week?

**Reply by StevenClontz** [reply]
Done! Thanks for the catch
